Source/Scene/Cesium3DTileContentType.js

```diff
@@ -0,0 +1,131 @@
+/**
+ * An enum to indicate the the different types of {@link Cesium3DTileContent}.
+ * For binary files, the enum value is the magic number of the binary file
+ * unless otherwise noted. For JSON files, the enum value is a unique name
+ * for internal use.
+ *
+ * @enum {String}
+ * @see Cesium3DTileContent
+ *
+ * @private
+ */
+var Cesium3DTileContentType = {
+  /**
+   * A Batched 3D Model. This is a binary format with
+   * magic number <code>b3dm</code>
+   *
+   * @type {String}
+   * @constant
+   * @private
+   */
+  BATCHED_3D_MODEL: "b3dm",
+  /**
+   * An Instanced 3D Model. This is a binary format with magic number
+   * <code>i3dm</code>
+   *
+   * @type {String}
+   * @constant
+   * @private
+   */
+  INSTANCED_3D_MODEL: "i3dm",
+  /**
+   * A Composite model. This is a binary format with magic number
+   * <code>cmpt</code>
+   *
+   * @type {String}
+   * @constant
+   * @private
+   */
+  COMPOSITE: "cmpt",
+  /**
+   * A Point Cloud model. This is a binary format with magic number
+   * <code>pnts</code>
+   *
+   * @type {String}
+   * @constant
+   * @private
+   */
+  POINT_CLOUD: "pnts",
+  /**
+   * TODO: description
```

Comment:
Bump to address the TODOs for vctr and geom

Reply:
I updated this with a rather general description of them, but let me know if there are important details I should include here.